src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/table.html.twig

```diff
@@ -24,6 +24,14 @@
  *#}
 
 {{ form_start(grid.filter_form, {'attr': {'id': grid.id ~ '_filter_form', 'class': 'table-responsive'}}) }}
+
+{{ renderhook('displayAdminGridTableBefore', {
```

Comment:
Should be outside the form. What if you want to include your own form inside it? :thinking:

Reply:
Makes sense

Reply:
:+1: I approve with @PierreRambaud